docs/reference/Custom Resources/certificate-stores.md

```diff
@@ -46,27 +49,30 @@ status:
       version:           yourCertVersion 
 ```
 
-| Name        | Required | Description | Default Value |
-| ----------- | -------- | ----------- | ------------- | 
-| vaultURI      | yes    |      URI of the azure key vault        |   ""            |
-| certificateName      | yes    |    the name of the key vault object   |       ""        |
-| certificateVersion   | no     |   provider will fetch latest version if empty   |     ""    |
-| tenantID   | yes     |   tenantID of the workload identity that have read access to this key vault   |     ""    |
-| clientID   | yes     |   clientID of the workload identity that have read access to this key vault   |     ""    |
+| Name               | Required | Description                                                               | Default Value |
```

Comment:
this is nice. Curious, does vscode extension auto format this?

Reply:
yes. I use this extension: https://marketplace.visualstudio.com/items?itemName=TakumiI.markdowntable